src/pages/extending-chat-widget/javascript-api/v1.0/index.mdx

```diff
@@ -7,6 +7,7 @@ tagline: "Customize the behavior of the Chat Widget."
 desc: "Adjust the mechanics of the widget or leverage the JavaScript API to pass additional details on to the Customer."
 versionGroup: "chat-widget"
 apiVersion: "1.0"
```

Comment:
This document can be indexed because it ha significantly different content than other versions. It's often the case for the oldest version